src/commands/open.js

```diff
@@ -61,6 +61,7 @@ var CHOICES = _.map(LINKS, "name");
 module.exports = new Command("open [link]")
   .description("quickly open a browser to relevant project resources")
   .before(requirePermissions)
+  .before(requireInstance)
```

Comment:
You need to add the require to the top, no?

Reply:
:man_facepalming: